third_party/move/tools/move-package/tests/test_sources/compilation/std-lib-conflicts/Move.exp

```diff
@@ -1 +1,53 @@
-Unable to resolve packages for package 'std-lib-conflicts': While resolving dependency 'B' in package 'std-lib-conflicts': Unable to resolve package dependency 'B': While resolving dependency 'AptosFramework' in package 'B': Unable to resolve package dependency 'AptosFramework': Conflicting dependencies found: package 'AptosFramework' conflicts with 'AptosFramework'
+CompiledPackageInfo {
```

Comment:
Hmm, thats weird. Can this be related to the issue we have with @fEst1ck PR? How could a failing test be in main, is it not running in CI?

Reply:
@fEst1ck added this test but @brmataptos found out that the output is flaky. Instead of updating the baseline, @fEst1ck also suggests we remove this test first and add another one with a stable output.